images/gpu-feature-discovery/tests/README.md

```diff
@@ -0,0 +1,197 @@
+# Notes on how this image was tested
```

Comment:
Thanks for adding, can we rename this to 'TESTING.md' and have it under /images/gpu-feature-discovery (i.e up one level?

That lines up with a convention we've been following

Reply:
don't forget to delete this one :)

Reply:
@mamccorm I just messed-up my git branch. [Made another PR](https://github.com/chainguard-images/images/pull/2688)